.eslintcache

```diff
@@ -0,0 +1 @@
+[{"C:\\Projects\\svelte-seo\\src\\lib\\index.svelte":"1","C:\\Projects\\svelte-seo\\src\\lib\\openGraph.svelte":"2","C:\\Projects\\svelte-seo\\src\\routes\\+page.svelte":"3"},{"size":3329,"mtime":1668633882001,"results":"4","hashOfConfig":"5"},{"size":3056,"mtime":1668633882671,"results":"6","hashOfConfig":"5"},{"size":501,"mtime":1668633882821,"results":"7","hashOfConfig":"5"},{"filePath":"8","messages":"9","suppressedMessages":"10","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"1bj33z1",{"filePath":"11","messages":"12","suppressedMessages":"13","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"14","messages":"15","suppressedMessages":"16","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"C:\\Projects\\svelte-seo\\src\\lib\\index.svelte",[],[],"C:\\Projects\\svelte-seo\\src\\lib\\openGraph.svelte",[],[],"C:\\Projects\\svelte-seo\\src\\routes\\+page.svelte",[],[]]
```

Comment:
please add this file to .gitignore

Reply:
Done

Reply:
Could you please delete this file from the repo, since it is in gitignore?

Reply:
Could you please delete this file from the repo, since it is in gitignore?

Reply:
@artiebits , it has been deleted